identifiers/country-tw/centrally_governed_level.csv

```diff
@@ -1,25 +1,23 @@
 id,name
 ocd-division/country:tw/province:cha,Changhua
 ocd-division/country:tw/province:cyq,Chiayi
```

Comment:
I think it might be a bit clearer to use the disambiguating term in in both names, so for this Chiayi County

Reply:
Yeah, you're right !